.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,35 @@
+name: 💡 Feature Request
+description: Suggest a new idea
+title: '[Feature Request]:'
+labels:
+  - needs triage
+  - enhancement
+body:
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your request related to a problem you have?
+      description: >-
```

Comment:
Maybe a typo?

Reply:
@luispfgarces it seems correct rendered [here ](https://github.com/Farfetch/kafkaflow-retry-extensions/blob/c47326ee0de8d69df8b0652ffe94fde86d6a30ec/.github/ISSUE_TEMPLATE/feature_request.yml)